code/controllers/subsystem/ticker.dm

```diff
@@ -75,6 +77,9 @@ SUBSYSTEM_DEF(ticker)
 			if(start_immediately)
 				time_left = 0
 
+			if(time_left <= 90 SECONDS && time_left > 60 SECONDS && !gamemode_vote)
+				addtimer(CALLBACK(SSvote, /datum/controller/subsystem/vote.proc/initiate_vote, "gamemode", "SERVER"), 1 SECONDS)
```

Comment:
what is changing gamemode_vote = TRUE to stop this spawning a shitton of timers?

Reply:
you raise a good point, and i dont have a good way to address this PR and dont think its worth it just yet (as an admin can do the votes)

gonna close for now and we can revisit it later if needed.